ros2cli/ros2cli/node/direct.py

```diff
@@ -53,10 +54,31 @@ def timer_callback():
     def __enter__(self):
         return self
 
+    # TODO(hidmic): generalize/standardize rclpy graph API
+    #               to not have to make a special case for
+    #               rclpy.action
+    def get_action_names_and_types(self):
+        return rclpy.action.get_action_names_and_types(self.node)
+
+    def get_action_client_names_and_types_by_node(
+        self,
+        remote_node_name,
+        remote_node_namespace
+    ):
+        return rclpy.action.get_action_client_names_and_types_by_node(
+            self.node, remote_node_name, remote_node_namespace)
+
+    def get_action_server_names_and_types_by_node(
+        self,
+        remote_node_name,
+        remote_node_namespace
+    ):
+        return rclpy.action.get_action_server_names_and_types_by_node(
+            self.node, remote_node_name, remote_node_namespace)
+
     def __getattr__(self, name):
         if not rclpy.ok():
             raise RuntimeError('!rclpy.ok()')
-
```

Comment:
Nitpick: unrelated whitespace change.

Reply:
Done in https://github.com/ros2/ros2cli/pull/421/commits/e3eee5c56a2faef4664f54289df965f94448bb13.